d2lbook/sphinx_template.py

```diff
@@ -52,8 +52,6 @@
 
 rsvg_converter_args = ['-z', '0.8']
 
-bibtex_bibfiles = ["BIBFILE"]
```

Comment:
Why do we need to remove this under `sphinx_conf`?

Reply:
It might be a sync issue.

Reply:
I'm adding it back per https://github.com/d2l-ai/d2l-book/commit/89db4fed7351ee289f63ea60f8baee7d1fd1cd87